File: Fundamentos de Programación/Ejercicio 2.6.py
a = input('Ingrese su número: ')
c = 1
while c<4:
    b = input('Ingrese su siguiente número: ')
    if a>=b:
        c = c+1
    else:
        a = b
        c = c+1
print('El número mayor es ' + str(a))

File: Fundamentos de Programación/Ejercicio 2.6.cpp
#include <iostream>

int main()
{
    int a = 0, b = 0, c = 1;
    std::cout<<"Ingrese su número: \n";
    std::cin >>a;
    
    while(c<4)
    {
    std::cout<<"Ingrese su número: \n";
    std::cin >>b;
        if(a>=b)
        {
            c++;
        }
        else
        {
            a = b;
            c++;
        }
    }
    std::cout<<"El número mayor es: "<<a<<"\n";
}